inst/rmarkdown/templates/acs_article/resources/template.tex

```diff
@@ -63,6 +63,14 @@
 $for(header-includes)$
 $header-includes$
 $endfor$
+$for(include-before)$
```

Comment:
These headers actually belong within the {document} around the `$body$` (and perhaps other LaTeX that goes at the beginning or end). See https://github.com/jgm/pandoc-templates/blob/master/default.latex#L231-L272